gontrib/contributions.go

```diff
@@ -44,7 +45,11 @@ func scanGit(project Project, emails []string, contributions []Contribution) (in
 	for _, repo := range project.Gitrepos {
 		util.PrintInfo("Working on "+repo, util.PI_TASK)
 		if PullSources {
-			git.GetLatestRepo(repo)
+			err := git.GetLatestRepo(repo)
+			if err != nil {
+				util.PrintInfo("Problem loading repo", util.PI_MILD_ERROR)
+				return 0, errors.New("Could not clone repo")
```

Comment:
sorry i only spotted this now.
since we write out the message via the util.PrintInfo we dont need a new one via errors.New, just pass the err from line 48 here